canvas/fizz-27-py/src/lib/canvasRequest.py

```diff
@@ -0,0 +1,10 @@
+import requests
+import os
+
+def get(url):
```

Comment:
Consider adding retries here. I'm using https://github.com/channable/opnieuw and it's quite easy to mix in. Just a function decorator.

Reply:
Retries is an interesting point and might make it worthwhile to fork the [canvasapi](https://github.com/ucfopen/canvasapi) package or continue with our own code, narrowly scoped to what we need.

One of the benefits of `canvasapi` is that it has a large test suite already. Also has built-in support for working with pagination.